tests/procfs/Makefile

```diff
@@ -25,4 +27,4 @@ myst:
 	$(MAKE) -C $(TOP)/tools/myst
 
 clean:
-	rm -rf $(APPDIR) rootfs export ramfs
+	rm -rf $(APPDIR) rootfs export ramfs *.o
```

Comment:
Is export a file name?

Reply:
Thanks for catching this. This probably crept in at some point.